validate.php

```diff
@@ -219,6 +219,8 @@
     $cmd = rtrim(shell_exec("which {$config[$bin]} 2>/dev/null"));
     if (!$cmd) {
         print_fail("$bin location is incorrect or bin not installed");
+        print_fail("You can also manually set the path to $bin by placing the following in config.php:");
+        print_fail("\$config['$bin'] = \"/path/to/$bin\";");
```

Comment:
Disregard this next pull. I think you want me to add all missing $bin into an array and then format output.... it that correct?